### PR TITLE
properly running calls should not show as 'Still running when workflow ended' [risk: no]

### PR DESCRIPTION
**Pull request by davidangb**
Fallout from FC's timing diagram customization in #1438: the timing diagram for running calls shows a tooltip of "Still Running when workflow ended", when it should show just "Running".

DataBiosphere/firecloud-app#234

-----

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
one failure, unrelated to timing diagram, which I will re-run manually:
```
[info] Tests: succeeded 128, failed 1, canceled 0, ignored 7, pending 0
[info] *** 1 TEST FAILED ***
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.workspace.WorkspaceOwnerSpec
[error] (Test / testOnly) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 1393 s, completed Dec 11, 2018 9:32:16 PM
```

EDIT: manual run passed in https://fc-jenkins.dsp-techops.broadinstitute.org/view/Testing/view/Pipelines/job/swatomation-pipeline/13993/

File: src/js/timingDiagram.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ function parseMetadata(data, dataTable, ancestry) {
                 executionCallsCount++;
                 // DA: added the "or now()" safeguard for anything still in Starting or otherwise doesn't have a workflowEnd value, for FireCloud.
                 var endDate = workflowEnd || new Date(Date.now());
-                if(endDate == null) {
+                if(workflowEnd == null) {
```
Comment (davidangb) [inline src/js/timingDiagram.js]: in #1438 I changed line 118 (see the  comment)  and ensured that `endDate` was non-null: https://github.com/broadinstitute/firecloud-ui/pull/1438/files#diff-ec6f318f92bf7a1ac08a023a5ba64862R117 This chage broke this `if` clause. Here, I'm returning the `if` clause to its original intent.

File: src/js/timingDiagram.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ function parseMetadata(data, dataTable, ancestry) {
                 executionCallsCount++;
                 // DA: added the "or now()" safeguard for anything still in Starting or otherwise doesn't have a workflowEnd value, for FireCloud.
                 var endDate = workflowEnd || new Date(Date.now());
```
Comment (breilly2) [inline src/js/timingDiagram.js]: Maybe move this `var` inside the `else` where it's used, or just inline it since it's now only used once.

**Review by breilly2 — APPROVED** [review]
👍 One suggestion but not critical.

File: src/js/timingDiagram.js (hunk 2 above)
Comment (davidangb) [reply src/js/timingDiagram.js]: thanks - this is a good suggestion and it led me to investigate further.  In the end I realized the tweak specific to FireCloud wasn't necessary at all - so I just removed it and reverted to using the exact same code Cromwell does (see https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/resources/workflowTimings/workflowTimings.html).

**Pull request merged**